README.md

```diff
@@ -179,13 +179,18 @@ The system supports two global deployment modes controlled by the `DEPLOY_ENV` i
 - **`LOCAL`** (default): Full local deployment with middleware contracts
 - **`TESTNET`**: Skip middleware deployment (assumes contracts already deployed on testnet)
 
-Projects can optionally use **MOCK** middleware deployments to speed up development.  
+Projects can configure different middleware deployment modes to speed up development or use different authentication mechanisms.
 This configuration should be set in the project's **Taskfile** variables:
 
 ```yaml
-USES_MOCK: true
+MIDDLEWARE_MODE: MOCK    # Options: ECDSA (default), MOCK, POA
 ```
 
+Available modes:
```

Comment:
@reecepbcups your call - but I'm not sure these naming conventions make sense, I believe we can theoretically have:

* Eigen with ECDSA/BLS
* Mock with ECDSA/BLS
* PoA with ECDSA/BLS

i.e. the signing mechanism is orthogonal to the consensus and reward/punishment mechanism